components/NewOutivityForm/index.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function NewOutivityForm({
+  CreateOutivity,
+  handleCancel,
+  selectedImage,
+}) {
+  return (
+    <main>
+      <StyledNewOutivitiesForm onSubmit={CreateOutivity}>
+        <h1>New Outivity</h1>
+        <StyledNewOutivitiesFormFields>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="image">Image</label>
+            {selectedImage && (
+              <Image
+                src="/{selectedImage}"
+                alt="outivity-image"
+                width={300}
+                height={200}
+              />
+            )}
```

Comment:
after seeing your whole application, I think you can remove these parts of your code:

```suggestion
```